crates/vision/src/image_segmenter.rs

```diff
@@ -580,15 +563,15 @@ mod tests {
         let scan_line = new_vertical_scan_line(
             &image,
             &field_color,
-            0,
+            1,
```

Comment:
Why did you change this value?

Reply:
If we use the first column in the image, we can't take a horizontal median of three as the sample to the left would be at x = -1.